### DISCUSSION TITLE
Updating Wiki

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#62] [Q&A] [answered]
I sometimes find it difficult to find the appropriate details for topics/concepts when reading the Wiki and have thought that it would be very useful to have links right where a topic is introduced/mentioned in the excellent overview README. Rather than just ask for such a thing and add to your workload, I thought I would try to contribute in a small way. I am pretty new to Github so as a test case, tried to make one such edit to the 'What else can sdm do?' section, adding a link to the Wiki page that explains appending lines to /etc/fstab.  If it's useful and properly done, I can add more such links as I work through my setup and use of sdm.  

To try this, I created a branch, edited and committed README.md on that branch and then submitted a pull request (https://github.com/dshaw619/sdm/pull/1).  That may not have been the proper procedure to bring it to your attention for possible inclusion in the master branch.  Let me know if this would be helpful and any advice on the best process.

**Comment by gitbls** [comment, marked as answer]
Um, I'm not seeing a pull request. I'm not what you'd call a github or git expert. It's one of those things I should learn more, but there's always something else... So, I'm not prepared to take pull requests at the moment.

That said, I definitely agree that the "What else..." section should have links to the wiki. Somehow that fell off my ToDo list. It's back on  and I'll take care of it.

**Reply by dshaw619** [reply]
My pull request is probably only to me! (:-)  I'll just submit any suggestions via the Discussions (and try not to overburden you).

**Reply by tiefenauer** [reply]
@dshaw619: You're right, your PR currently only affects your own repo (dshaw619/sdm), which is a fork of this one. If you want your changes to be merged back into the repo from which you forked (this repo here), you have to set the target branch accordingly. It is currently set to `master` but it needs to be `gitbls:master`.

You can read more about how creating a PR from a fork works here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

Note: Technically, both the `master` branch on your fork and on this repo are just that: branches, but with a different origin.

**Reply by dshaw619** [reply]
@https://github.com/tiefenauer:  Thanks for that explanation and link.  I'll read it through.
